internal/publish/config.go

```diff
@@ -43,8 +43,8 @@ type Config struct {
 	TruncateWindow     time.Duration `envconfig:"TRUNCATE_WINDOW" default:"1h"`
 
 	// Flags for local development and testing.
-	DebugAPIResponses   bool `envconfig:"DEBUG_API_RESPONSES"`
-	DebugAllowRestOfDay bool `envconfig:"DEBUG_ALLOW_REST_OF_DAY"`
```

Comment:
We'll need to update the Terraform configurations for the server instances with this and coordinate a deploy.

Reply:
this shouldn't be in the terraform - as we want the default value of false.

I set the old value manually in our UAT environment.